### ISSUE TITLE
Custom Tree Indent too small on HiDPI

### ISSUE THREAD

**Issue by jaspernbrouwer** [ChrisRM/material-theme-jetbrains#697]
The space between the folder-arrows and -icons is too small IMHO. Without "Custom Tree Indent" checked they appear to be touching, with "Custom Tree Indent" checked and set to max (10) they are stil very close to each other.

### "Custom Tree Indent" unchecked

![tree-unchecked](https://user-images.githubusercontent.com/738933/37904793-151f1ca6-30fd-11e8-8621-91e35fce4847.png)

As you can see the arrows are _almost_ touching the icons. These are HiDPI screenshots, so when scaled they appear as touching.

### "Custom Tree Indent" checked and set to max (10)

![tree-checked-max](https://user-images.githubusercontent.com/738933/37904807-1a525fa8-30fd-11e8-8dfa-5cc64dd59869.png)

The arrows aren't touching the icons anymore, but they're still awfully close IMO. Especially when the folder is expanded (arrow pointing down).

### "Settings"

![settings](https://user-images.githubusercontent.com/738933/37904973-ac551f76-30fd-11e8-9040-9b706cc062f0.png)

Here in the settings they appear to have more breathing room. This is beginning to look better IMO.

### A thought

Might this be related to HiDPI screens?

I can imagine that HiDPI or not, 1px is 1px. But on HiDPI screens that pixel appears as half a pixel, because of scaling :wink: 

### Environment

PhpStorm 2017.3.6
Build #PS-173.4674.46, built on March 15, 2018
JRE: 1.8.0_152-release-1024-b15 amd64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
Linux 4.13.0-37-generic
Material Theme UI 1.6.1

```
$ cat .idea/material-theme-jetbrains.iml 
<?xml version="1.0" encoding="UTF-8"?>
<module type="WEB_MODULE" version="4">
  <component name="NewModuleRootManager">
    <content url="file://$MODULE_DIR$" />
    <orderEntry type="inheritedJdk" />
    <orderEntry type="sourceFolder" forTests="false" />
  </component>
</module>
```

**Comment by jaspernbrouwer** [comment]
This is a duplicate of #648, but that issue was closed without being fully addressed.

**Comment by mallowigi** [comment]
I found out that the directory icons are not correctly centered, maybe it’s the reason

**Comment by jaspernbrouwer** [comment]
Yes, I see centering could help. I do think there's still not enough room, it seems a bit squashed together IMO (looking at my screenshots). I also feel it's not what's intended, but not sure on that.

I went ahead and made a PR to increase the max to 20, as a quick-fix!

**Comment by jaspernbrouwer** [comment]
Sorry, what didn't work precisely?

**Comment by mallowigi** [comment]
The change you submitted

**Comment by mallowigi** [comment]
Ok I fixed it in the latest version (2.1.6, available soon)
